src/main/java/org/edumips64/utils/cli/FileCommand.java

```diff
@@ -0,0 +1,61 @@
+package org.edumips64.utils.cli;
+
+import org.edumips64.core.CPU;
+import org.edumips64.core.parser.ParserMultiException;
+import org.edumips64.utils.CurrentLocale;
+import org.edumips64.utils.io.ReadException;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.Option;
+
+import java.io.File;
+
+
+@Command(name = "file", resourceBundle = "CliMessages")
```

Comment:
I'd suggest to rename `FileCommand` to `LoadCommand`, since "load" to me is the action that the command performs. (or "open", maybe?)

Reply:
I think load sounds good. Changed to that.